dindex_store/common.py

```diff
@@ -49,6 +49,26 @@ def add_undirected_edge(self, source: int, target: int, type: EdgeType, weigth:
         :return:
         """
         pass
+    
+    @abstractmethod
+    def find_neighborhood(self, node, hops=2):
```

Comment:
cannot think of a use case where we are interested in a neighborhood more than one hop away, but I think this is a good generalization. I'd change hops to 1 by default though